src/payment/__factories__/basket.factory.js

```diff
@@ -31,6 +30,7 @@ Factory.define('basket')
     }
     return products;
   })
+  .attr('summary_quantity', ['numProducts'], numProducts => numProducts)
```

Comment:
This needed to match the number of products.

Reply:
Really? I thought quantity was the number of enrollment codes for a product?